data/scripts/dependency-resolver.py

```diff
@@ -141,7 +155,8 @@ def compile_test(source, compiler, cflags, ldflags):
     output = "%s-bin" % f.name
     cmd = "{compiler} {cflags} {src} -o {out} {ldflags}".format(compiler=compiler,
             cflags=cflags, ldflags=ldflags or "", src=f.name, out=output)
-    out, status = run_command(cmd)
+    context.add_log_content("Compiling source code:\n%s" % source)
```

Comment:
is this printing the actual source or just the temporary location? It would be nice to print the source.